packages/issuer-sdk-js/src/credentials.ts

```diff
@@ -42,16 +41,15 @@ const buildInsertableIDOSCredential = (
     publicNotes,
     plaintextContent,
     recipientEncryptionPublicKey,
-    contentHash,
   }: {
     userId: string;
     publicNotes: string;
     plaintextContent: Uint8Array;
     recipientEncryptionPublicKey: Uint8Array;
-    contentHash?: string;
   },
+  encryptKeyPair?: nacl.BoxKeyPair,
```

Comment:
Why are we reusing the ephemeral keypair between the original and the copy credential? :thinking:

Reply:
Because we use [one encryptor key](https://github.com/idos-network/idos-schema/pull/68/files#diff-cac5209c47f01e07281d9786c08ca81379320cd777d1ff1fb4d9bbf133326e80R616) for original and copy . Otherwise we have to add one more argument to the `create_credentials_by_dwg` action.

Reply:
Returned back.